internal/pkg/project/manifest/file.go

```diff
@@ -14,7 +14,10 @@ import (
 )
 
 const (
-	FileName = "manifest.json"
+	FileName                      = "manifest.json"
+	DefaultTemplateRepositoryName = "keboola"
+	DefaultTemplateRepositoryUrl  = "git@github.com:keboola/keboola-as-code-templates.git"
+	DefaultTemplateRepositoryRef  = "main"
```

Comment:
Nedame to do `internal/pkg/template/repository/default.go`? 
+ nejaku funkciu `DefaultRepository()`, ktoru by si pouzil nizsie.

Uklada sa to do projektu, ale su to informacie o repe.
Co myslis?

Reply:
jo to zní dobře 👍